docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[embedding]]
+== Embedding {kib}
+
+Once you create a dashboard, or a visualization you surely might want to share it with your colleagues or friends at some point. The easiest way to do that is to share a direct link to your dashboard or visualization, but not everyone might have access to your {kib} or even know what {kib} is. Moreover, sometimes you might want to give more context to the thing you're sharing, for example, attach analysis, or images, or links, and {kib} isn't always the best tool for this type of job.
+
+If you have an internal company website or just a personal webpage where it'd make more sense to display the content you created in {kib}, then try out the {kib} embedding functionality. You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in the *Share > Embed code* menu in *Dashboard* or *Visualize*.
```

Comment:
I like the new content!  Here is a suggestion that moves things around a bit and removed some of the things that felt a little negative.  In the second para, I suggest leading with the Kibana embedding functionality and putting the instructions in the order that that users sees them.  Also, the part about adding context felt more natural here.

> Once you create a dashboard or a visualization, you might want to share it with your colleagues or friends. The easiest way to do this is to share a direct link to your dashboard or visualization.  However, some users might not have access to your {kib}. 

> With the {kib} embedding functionality, you can display the content you created in {kib} to an internal company website or a personal web page.  From *Dashboard* or *Visualize*, open the *Share > Embed code* menu, and then click *Copy iFrame code* to generate an HTML code snippet.  You can embed in this snippet in your web page, and then add analysis, images, and links to give more context to the object you're sharing.

Reply:
Sounds much better, thanks!